linkedlist.mod/doc/clearlist.bmx

```diff
@@ -0,0 +1,22 @@
+SuperStrict
+
```

Comment:
While I understand the approach of using `Framework` I would avoid it in the "procedural interface" examples. 
It should concentrate on the individual functions instead of trying to contain other useful information.

Reply:
So where do you think is the best place to introduce users to good practice, other than in the provided examples?

Reply:
As said I understand the approach - but I would introduce them within more complex examples - eg within a "how to write a game loop" or so. Or within tutorials.

I think it might frighten people if you put 10 lines of code around a simple `print "Hello World"`. Even the `SuperStrict` isn't really needed (while it is good to use it).


So this is why I thought that we might "mix" stuff: procedural interfaces are kept "beginner friendly" while the "OOP examples" are for more experienced people - and contain `SuperStrict`, ` Framework` ... and maybe even other stuff I cannot think of now.

Reply:
Using the `Framework` command might sound intriguing when it comes to improve your coding level. And I understand you in the regards that having them in the examples helps to teach people "what to import if you need XYZ".
Still using `Framework` will lead to many situations for beginners in which stuff then just does not work anymore. They forget to `Import Brl.PNGLoader` (while they imported Max2D and so on) and then wonder why `LoadImage("img.bmp")` works but it then fails for "img.png".

This is what I mean with "beginner friendly".

Reply:
I created another Pull Request containing the desired Framework+Import statement - so once you think discussion is finished/over/done you can accept the one you favor - and reject the other one.